src/test/kotlin/graphics/scenery/tests/examples/compute/PersistentTextureRequestsExample.kt

```diff
@@ -108,33 +118,16 @@ class PersistentTextureRequestsExample : SceneryBase("PersistentTextureRequestsE
                 //the buffer can now, e.g., be transmitted, as is required for parallel rendering
 
                 if (buffer != null && prevCounter == 100) {
-                    SystemHelpers.dumpToFile(buffer, "texture-${SystemHelpers.formatDateTime(delimiter = "_")}.raw")
-                }
-            }
-        }
+                    val file = SystemHelpers.dumpToFile(buffer, "texture-${SystemHelpers.formatDateTime(delimiter = "_")}.raw")
+                    assertNotNull(file, "File handle should not be null.")
 
-        thread {
-            while (renderer?.firstImageReady == false) {
-                Thread.sleep(5)
+                    val sum = file.readBytes().sum()
+                    logger.info("Dumped: $file, ${file.length()} bytes, sum=$sum")
+                    assertEquals(file.length(), 1280*720*4, "File should contain the correct number of bytes for a 1280x720xRGBA image")
+                    assert(sum > 3000000) { "Sum of bytes in file should be non-zero" }
```

Comment:
Why is the sum compared with 3000000? Is it not sufficient to check that it is not equal to zero? Changes in volume rendering code down the line could change the image generated and therefore the sum